tools/alpine/packages

```diff
@@ -87,12 +87,13 @@ open-iscsi
 openntpd
 openrc
 openresolv
+openssh-client
 openssh-server
 openssl
 openssl-dev
 patch
 pigz
-python
+python2
```

Comment:
Hmm, no we should not install python 2 it is out of support

Reply:
What requires it if anything?

Reply:
This is a good question. Do we depend on it anywhere? I was trying to make as little of a change as possible, in this case making sure that python2 was where it was expected

Reply:
```
grep -r python . 2>/dev/null | grep -v -w vendor | grep -v 'tools\/alpine'
./tools/grub/Dockerfile:  python3 \
./tools/grub/Dockerfile:# because python is not available
./tools/grub/Dockerfile:RUN ln -s python3 /usr/bin/python
./projects/miragesdk/examples/https-unikernel/Dockerfile:RUN sudo apt-get install -y screen python-pip python-setuptools python-dev --no-install-recommends
./projects/miragesdk/examples/https-unikernel/README.md:$ python
./projects/miragesdk/examples/https-unikernel/README.md:(or just do `python src/test_store.py`)
./projects/memorizer/docs/scripts/README.txt:Ex: python memorizer.py "ls" "mkdir dir"
./projects/memorizer/docs/scripts/README.txt:Ex: python test_memorizer.py -e
./scripts/kconfig-split.py:#!/usr/bin/env python
./kernel/Dockerfile.bcc:  python \
./kernel/Dockerfile.bcc:RUN mkdir -p /out/usr/lib/python2.7 && \
./kernel/Dockerfile.bcc:    cp -a /usr/lib/python2.7/site-packages /out/usr/lib/python2.7/
./kernel/Dockerfile.bcc:  python \
./kernel/Dockerfile.perf:    python3 \
```

So:

* `tools/grub` uses python3, so that's fine
* `projects/miragesdk/examples/https-unikernel/Dockerfile` uses its own `FROM`, so it doesn't count
* `kernel/Dockerfile.perf` uses python3, so that's fine
* `scripts/kconfig-split.py` is used on the developer platform, so that's fine
*  `kernel/Dockerfile.bcc` uses python

Basically only `kernel/Dockerfile.bcc`. @rn might know?